src/main/java/com/javaproject/restservice/Application.java

```diff
@@ -0,0 +1,11 @@
+package com.javaproject.restservice;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
```

Comment:
What does this annotation do?

Reply:
it allows me to run the application as a web service in a stand-alone server.